web/src/modules/common/components/Tooltip.js

```diff
@@ -1,41 +1,28 @@
 // @flow
 
 import * as React from 'react'
-import { useContext } from 'react'
 import styled, { css } from 'styled-components'
 import dimensions from '../../theme/constants/dimensions'
-import PlatformContext from '../../platform/PlatformContext'
 import type { StyledComponent } from 'styled-components'
 import type { ThemeType } from 'build-configs/ThemeType'
 
-// Works for Chrome > 69, Firefox > 41, RTL/LTR support does not work for IE
-
-const toLogicalProperty = (prop: string, supportsLogicalProperties: boolean): string => {
-  if (!supportsLogicalProperties) {
-    return prop
-  }
-
-  switch (prop) {
-    case 'right':
-      return 'inset-inline-end'
-    case 'left':
-      return 'inset-inline-start'
-    case 'border-left-width':
-      return 'border-inline-start-width'
-    case 'border-right-width':
-      return 'border-inline-end-width'
-    case 'border-left-color':
-      return 'border-inline-start-color'
-    case 'border-right-color':
-      return 'border-inline-end-color'
+// The previous implementation of this function used css logical properties. Therefore the name
```

Comment:
Otherwise it looks like the comment is incomplete.
```suggestion
// The previous implementation of this function used css logical properties. Therefore the name.
```